src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage } from 'react-intl';
 import Loading from '../../Loading/Loading';
-import './ReportsPageRanking.css';
```

Comment:
Remember also delete the .css file

Reply:
completely eliminate?

Reply:
Of course!

Reply:
yes it's only create for the initial styles, no needed